File: StudentSystem.Models/Faculty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace StudentSystemDbContext.Models
{
    public class Faculty
    {
        public int Id { get; set; }

        public String Name { get; set; }

        public IEnumerable<Specialty> Specialties { get; set; }

        public IEnumerable<Student> Students { get; set; }

        public IEnumerable<Lecturer> Lecturers { get; set; }
    }
}

File: db/models/Faculty.js

module.exports = function (sequelize, DataTypes) {
    const Sale = sequelize.define('Faculty', {
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        name: {
            type: DataTypes.STRING
        },
        specialties: {
            type: DataTypes.STRING
        },
        students: {
            type: DataTypes.STRING
        },
        lecturers: {
            type: DataTypes.STRING
        }
    });
    
    // Class Method
    Sale.associate = function (models) {
    };
    
    // Instance Method
    // Sale.prototype.someMethod = function () {
    // };

    return Sale;
};